src/headless/plugins/emoji/plugin.js

```diff
@@ -75,8 +75,30 @@ converse.plugins.add('converse-emoji', {
                 async initialize () {
                     if (!converse.emojis.initialized) {
                         converse.emojis.initialized = true;
+
                         const module = await import(/*webpackChunkName: "emojis" */ './emoji.json');
-                        const json = (converse.emojis.json = module.default);
+                        /**
+                         * *Hook* which allows plugins to modify emojis definition.
+                         *
+                         * Note: This hook is only fired one time, when ConverseJS initiliazed.
+                         *
+                         * @event _converse#loadEmojis
+                         * @param json
+                         *      See src/headless/emojis.json for more information about the content of this parameter.
```

Comment:
Would be nice if you used the JSDoc `@link` tag to make a hyperlink.

Reply:
Done.